Src/PerformanceCollector/Perf.Shared.Tests/QuickPulse/QuickPulseTelemetryProcessorTests.cs

```diff
@@ -2986,5 +2986,165 @@ public void QuickPulseTelemetryProcessorHandlesCalculatedMetricsInThreadSafeMann
             Assert.AreEqual((taskCount / 2) + 1, allBadSlowMinValue);
             Assert.AreEqual(taskCount - 1, allBadSlowMaxValue);
         }
+
+#if NET45
+        [TestMethod]
+        public void VerifyInitializationWhenDeferredIsTrue()
+        {
+            var config = new TelemetryConfiguration();
+            config.ExperimentalFeatures.Add("deferRequestTrackingProperties");
+
+            var spy = new SimpleTelemetryProcessorSpy();
+            var telemetryProcessor = new QuickPulseTelemetryProcessor(spy);
+            telemetryProcessor.Initialize(config);
+
+            Assert.IsTrue(telemetryProcessor.EvaluateDisabledTrackingProperties);
```

Comment:
why this test inside net45? The actual code is doing this irrespective of net45 or not.

Reply:
the code path i'm testing is NET45 only.

Reply:
product code does have NET45 condition.